src/UI.elm

```diff
@@ -1,4 +1,4 @@
-module UI exposing (..)
+module UI exposing (buttonAttrs)
```

Comment:
@SimonLab am I missing something? how does this PR update the status?

Reply:
At the moment the api repository is on my account.
I've created the PR here: https://github.com/SimonLab/app-api/pull/10/files

Also I think it would be better on a long term to move this repository to dwyl

Reply:
Thanks for clarifying. 👍